docs/guides/databases/mariadb/backup-mariadb-mysql-to-object-storage-with-restic/index.md

```diff
@@ -53,45 +55,59 @@ The steps in this guide require root privileges. Be sure to run the steps below
 
 ## Install Restic
 
-{{< content "applications/backups/install-restic-shortguide" >}}
+{{< content "install-restic-shortguide" >}}
```

Comment:
For these shortguide embeds, Hugo only expects to be passed the slug to the shortguide, and not the categories that it falls under. This also means that shortguides needs to have unique names. It's a little strange to me that it works this way, but 🤷 

In any case, an easy fix